logbook-servlet/src/main/java/org/zalando/logbook/servlet/RemoteRequest.java

```diff
@@ -117,13 +115,10 @@ public HttpRequest withBody() throws IOException {
     private boolean isFormRequest() {
         @Nullable final String contentType = getContentType();
 
-        if (contentType != null) {
-            final MimeType mimeType = MimeTypes.parse(contentType);
-            return "application".equals(mimeType.getPrimaryType()) &&
-                    "x-www-form-urlencoded".equals(mimeType.getSubType());
-        }
-
-        return false;
+        return contentType != null && MimeTypes.parse(contentType)
```

Comment:
You could do `Optional.ofNullable(contentType).map(MimeTypes::parse)` as well

Reply:
:man_facepalming: right...

Reply:
Done